app/components/common/userInput/index.test.tsx

```diff
@@ -1,53 +1,71 @@
 import { fireEvent, render, screen } from '@testing-library/react';
 import UserInput from '.';
 
-it('renders a input field', () => {
-  render(
-    <UserInput
-      label="Username"
-      placeholder="enter your name"
-      link="hap.day/"
-      value=""
-      setValue={() => {}}
-    />
-  );
-  const userInput = screen.getByTestId('user-input');
-  expect(userInput).toBeInTheDocument();
-});
+describe('UserInput', () => {
+  it('renders a input field', () => {
+    render(
+      <UserInput
+        label="Username"
+        placeholder="enter your name"
+        link="hap.day/"
+        value=""
+        setValue={() => {}}
+      />
+    );
+    const userInput = screen.getByTestId('user-input');
+    expect(userInput).toBeInTheDocument();
+  });
 
-it('checks added button in input field wrapper', () => {
-  render(
-    <UserInput
-      label="Username"
-      placeholder="enter your name"
-      link="hap.day/"
-      value=""
-      setValue={() => {}}
-    />
-  );
-  const userInputWrapper = screen.getByTestId('user-input-wrapper');
-  const button = screen.getByTestId('url-btn');
-  expect(userInputWrapper).toContainElement(button);
-});
+  it('checks added button in input field wrapper', () => {
+    render(
+      <UserInput
+        label="Username"
+        placeholder="enter your name"
+        link="hap.day/"
+        value=""
+        setValue={() => {}}
+      />
```

Comment:
Since you're rendering `UserInput` with same props, you can wrap it under `beforeEach()` and you can have a describe block to label the props.
Refer: https://reactjs.org/docs/testing-recipes.html#setup--teardown

Reply:
I am not passing the same props each time. Creating beforeEach and maintaining separate prop variable will be redundant work.